applications/FluidDynamicsApplication/python_scripts/navier_stokes_solver_vmsmonolithic.py

```diff
@@ -146,7 +146,8 @@ def CreateSolver(model, custom_settings):
 
 class NavierStokesSolverMonolithic(FluidSolver):
 
-    def _ValidateSettings(self, settings):
+    @classmethod
```

Comment:
Why does this need to be classmethod?

Reply:
It does not depend on an instance of the class